docker/1.5-1/final/Dockerfile.cpu

```diff
@@ -15,7 +15,9 @@ RUN python3 -m pip install -r /requirements.txt && rm /requirements.txt
 # Copy wheel to container #
 ###########################
 COPY dist/sagemaker_xgboost_container-2.0-py2.py3-none-any.whl /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl
-RUN rm -rf /miniconda3/lib/python3.7/site-packages/numpy-1.21.2.dist-info && \
+
+RUN conda install numpy --force-reinstall && \
```

Comment:
Why do we `force-reinstall` the numpy here?  Will it install the latest numpy version? Shouldn't we pin https://github.com/aws/sagemaker-xgboost-container/blob/master/requirements.txt#L11 here and use pinned numpy?

Reply:
Yes that is a good point. I'm not sure why the dockerfile got pushed. I'll remove it from the PR

Reply:
The change has been added!